rotkehlchen/inquirer.py

```diff
@@ -500,6 +516,161 @@ def find_uniswap_v2_lp_price(
         share_value = numerator / total_supply
         return Price(share_value)
 
+    def find_curve_pool_price(
+        self,
+        lp_token: EthereumToken,
+    ) -> Optional[Price]:
+        """
+        1. Obtain the pool for this token
+        2. Obtain prices for assets in pool
+        3. Obtain the virtual price for share and the balances of each
+        token in the pool
+        4. Calc the price for a share
+
+        Returns the price of 1 LP token from the pool
+        """
+        assert self._ethereum is not None, 'Inquirer ethereum manager should have been initialized'  # noqa: E501
+
+        pools = get_curve_pools()
+        if lp_token.ethereum_address not in pools:
+            return None
+        pool = pools[lp_token.ethereum_address]
+        tokens = []
+        # Translate addresses to tokens
+        try:
+            for asset in pool.assets:
+                # Eth address is translated to weth
+                if asset == '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE':
+                    tokens.append(A_WETH)
```

Comment:
Remove obvious comment

```suggestion
            for asset in pool.assets:
                if asset == '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE':
                    tokens.append(A_WETH)
```